qiita_pet/templates/list_analyses.html

```diff
@@ -8,8 +8,78 @@
     {% if analyses %}
       $('#no-analyses').hide()
       $('#analyses-table').dataTable({
-        "order": [[ 2, "desc" ]]
+        "order": [[ 2, "desc" ]],
+        columnDefs: [
+          {"render": function ( data, type, row, meta ) {
+            data = JSON.parse(data);
+            var len = 0;
+            if (data !== null && data !== undefined){
+              len = data.length;
+             }
+            if (len != 0){
```

Comment:
Minor, this could be short-circuited as: `if (data !== null && data !== undefined && data.length !=0)`